net-misc/scponly/scponly-4.8-r8.ebuild

```diff
@@ -0,0 +1,249 @@
+# Copyright 1999-2019 Gentoo Authors
```

Comment:
2020?

Also, why not -r7, when the latest version in the tree is -r6?

Reply:
Ah, because I have a -7 in my local overlay already (had already fixed the related bug, and then did the GLEP 81 changes separately) and I was lazy.  Will fix next submission.